1e/manifest.json

```
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d86565ae-ca60-45c2-a0b9-944ec1e05a15",
+  "app_id": "one-e",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your 1E products metrics to DataDog.",
+    "title": "1E Integration",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Category::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "1e",
```

```suggestion
      "source_type_name": "one-e",
```

for consistency with the `app_id`

---

Applying to speed up the process

---

This doesn't look like it got changed to "one-e"?